diff --git a/node_modules/universal-react-scripts/config/webpack.config.js b/node_modules/universal-react-scripts/config/webpack.config.js
index ba18248..68e8e1b 100644
--- a/node_modules/universal-react-scripts/config/webpack.config.js
+++ b/node_modules/universal-react-scripts/config/webpack.config.js
@@ -11,7 +11,6 @@
 const fs = require('fs');
 const isWsl = require('is-wsl');
 const path = require('path');
-const webpack = require('webpack');
 const resolve = require('resolve');
 const PnpWebpackPlugin = require('pnp-webpack-plugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
@@ -35,6 +34,8 @@ const ModuleNotFoundPlugin = require('universal-react-dev-utils/ModuleNotFoundPl
 const ForkTsCheckerWebpackPlugin = require('universal-react-dev-utils/ForkTsCheckerWebpackPlugin');
 const typescriptFormatter = require('universal-react-dev-utils/typescriptFormatter');
 const eslint = require('eslint');
+const webpack = require('webpack');
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
 // @remove-on-eject-begin
 const getCacheIdentifier = require('universal-react-dev-utils/getCacheIdentifier');
 // @remove-on-eject-end
@@ -71,6 +72,8 @@ module.exports = function(webpackEnv, executionEnv) {
   const isEnvProduction = webpackEnv === 'production';
   const isEnvNode = useNodeEnv && executionEnv === 'node';
   const isEnvWeb = executionEnv === 'web';
+  const isCI = process.env.CI === 'true';
+  const isAnalyze = process.env.ANALYZE
 
   // Variable used for enabling profiling in Production
   // passed into alias object. Uses a flag if passed into the build command
@@ -236,7 +239,11 @@ module.exports = function(webpackEnv, executionEnv) {
           // When performing server side rendering, the app should load modules
           // from node_modules instead of bundling them. This function take care
           // of making sure that any module in node_modules is treated as external.
-          nodeExternals(),
+          nodeExternals({
+            whitelist: [
+              /babel-plugin-universal-import|react-universal-component/
+            ]
+          }),
         ],
     optimization: {
       minimize: isEnvProduction,
@@ -437,6 +444,15 @@ module.exports = function(webpackEnv, executionEnv) {
                 name: 'static/media/[name].[hash:8].[ext]',
               },
             },
+            
+            // Makana Custom Loaders
+            {
+              test: /\.graphql$/,
+              exclude: /node_modules/,
+              loader: require.resolve('graphql-tag/loader'),
+            },
+            // End Makana Custom Loaders
+
             // Process application JS with Babel.
             // The preset includes JSX, Flow, TypeScript, and some ESnext features.
             {
@@ -623,6 +639,14 @@ module.exports = function(webpackEnv, executionEnv) {
       ],
     },
     plugins: [
+      // Makana Custom Additions
+      isAnalyze && new BundleAnalyzerPlugin(),
+      isCI && new webpack.ProgressPlugin(),
+      isEnvNode && new webpack.optimize.LimitChunkCountPlugin({
+        maxChunks: 1,                               
+      }),
+      // End Makana Custom Additions
+
       // Generates an `index.html` file with the <script> injected.
       isEnvWeb &&
         new HtmlWebpackPlugin(
diff --git a/node_modules/universal-react-scripts/scripts/start.js b/node_modules/universal-react-scripts/scripts/start.js
index 31d2e86..e9b1a97 100644
--- a/node_modules/universal-react-scripts/scripts/start.js
+++ b/node_modules/universal-react-scripts/scripts/start.js
@@ -34,8 +34,10 @@ verifyTypeScriptSetup();
 const fs = require('fs');
 const chalk = require('universal-react-dev-utils/chalk');
 const webpack = require('webpack');
+require('universal-react-dev-utils/clearConsole');
+const clearConsole = ()=>{}
+require.cache[require.resolve('universal-react-dev-utils/clearConsole')].exports = clearConsole
 const WebpackDevServer = require('webpack-dev-server');
-const clearConsole = require('universal-react-dev-utils/clearConsole');
 const checkRequiredFiles = require('universal-react-dev-utils/checkRequiredFiles');
 const {
   choosePort,
